### ISSUE TITLE
Edit Question: [ua_ruhr/storage-and-long-term-preservation/long-term-preservation-datasets/access_authentication]

### ISSUE THREAD

**Issue by jmfrenzel** [FDM-UARuhr/rdmo-catalog-uaruhr#34]
**Describe the request**  
Add help text 

**Propose what exactly should be changed**  
As suggested below, please choose from the list in Additional context and provide the updated values.

"Wie wird die Identität von Personen, die auf die Daten zugreifen, verifiziert?" Hilfetext wäre hier hilfreich:-) 

**Original setting or content**   
What is the current content or setting e.g. 

*question text en*  
"none"

**Proposed setting or content**   
What should it look like in the new version, e.g. 

*question text en*  
"to be disussed"

**Additional context**

**From the following list delete everything that is NOT part of the content of your proposal. Once a subject of change is approved it will be marked accordingly in the list.**  
- [ ] uri
- [ ] key
- [ ] path
- [ ] questionset
  - [ ] position in set
- [ ] question text (en and ger)
- [x] help text (en and ger)
- [ ] answer
  - [ ] widget type
  - [ ] value type
  - [ ] max/min, step and unit
- [ ] optionset
- [ ] condition
- [ ] attribute

**Comment by jmfrenzel** [comment]
move to rdmo-catalog

**Comment by jmfrenzel** [comment]
closed in favor of https://github.com/rdmorganiser/rdmo-catalog/issues/189
